src/test/java/roomescape/fixture/fixture.java

```diff
@@ -0,0 +1,37 @@
+package roomescape.fixture;
+
+import roomescape.domain.Member;
+import roomescape.domain.Role;
+import roomescape.domain.Theme;
+import roomescape.domain.TimeSlot;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+public class fixture {
```

Comment:
```suggestion
public class Fixture {
```
보통 Fixture 클래스는 각각의 도메인의 Fixture들을 가진 뭉치이기 때문에 하나의 `Fixture`에 두는 것이 아닌 `MemberFixture`로 관리하는 것이 그 목적에 좀 더 적절해요.

Reply:
변경: Fixture 분리를 완료하였습니다.

다만 현재처럼 public static ... 상수 모양으로 정의해놓고 가져다 쓰는 게 올바른 fixture 사용 방식이 맞는지가 궁금합니다!
fixture에 대해 조금 더 찾아보고 공부해보니 이와 다른 방식(ex. create() 메소드...)로 구현되고 있더라고요.

```java
@Component
public class MemberFixture {
    @Autowired
    private MemberRepository memberRepository;

    public Member createUserMember() {
        Member user = new Member(
                new MemberName("사용자"),
                new MemberEmail("user@gmail.com"),
                new MemberPassword("1234567890"),
                MemberRole.USER
        );
        return memberRepository.save(user);
    }

```

이런 느낌으로요!

혹시 제가 지금 구현해놓은 방식은 fixture라는 네이밍에 맞지 않는 방식인 걸까요...? 😭

Reply:
정의하기에 따라서 다를 것 같은데 개인적으로는 Fixture라고 보기 힘들 것 같아요.

Fixture는 자주 사용되는 데이터에 대한 정의이지 이렇게 생성하는 기능을 담는다고 보기는 어려워요. 만들어주신 static과 같은 상수의 모양으로도 충분해보이고, 혹은 enum을 활용해볼 수 도 있어요.